examples/project_llm_fine_tune/project_llm_fine_fune/assets.py

```diff
@@ -0,0 +1,395 @@
+import dagster as dg
+import pandas as pd
+from collections import defaultdict
+import time
+from dagster_openai import OpenAIResource
+import dagster_duckdb as dg_duckdb
+from . import utils
+import json
+from textwrap import dedent
+from collections import Counter
+from openai import OpenAI
+from typing import Iterable
+import project_llm_fine_fune.constants as constants
```

Comment:
ruff should sort these imports.

`from . import utils` I like to try and avoid relative imports